.remill_commit_id

```diff
@@ -1 +1 @@
-d5c50357287cae0072938477b3b525bafcce8069
\ No newline at end of file
+a8ead7b58496f0fc90100eee67de1eee74cdc8c3
```

Comment:
Side-note, since we're here... Why don't we use a submodule?

Reply:
Good point. How's this?